pipelines/broad/arrays/imputation_beagle/test_inputs/Plumbing/NA12878_x10_hg38_arrays.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ImputationBeagle.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/scientific/vcfs/NA12878_10_duplicate.merged.cleaned.vcf.gz",
```

Comment:
i see that this path points to a vcf.gz in a scientific directory, is that expected?

Reply:
that sounds great!